docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_constructions.md

```diff
@@ -0,0 +1,103 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_constructions.md"]
+```
+
+# Constructions
+
+The standard way to define a polyhedron is by either giving a V-description or an H-description.
```

Comment:
What's a H-/V-description? Is it the same as H-/V-representation? If yes, why two names? If no, what is what and how do they relate?

Also: perhaps $H$ and $V$ instead of H and V?